x-pack/plugins/siem/public/endpoint_alerts/view/details/overview/index.tsx

```diff
@@ -22,9 +23,13 @@ import { FormattedDate } from '../../formatted_date';
 import { AlertDetailResolver } from '../../resolver';
 import { ResolverEvent } from '../../../../../common/endpoint/types';
 import { TakeActionDropdown } from './take_action_dropdown';
+import { urlFromQueryParams } from '../../url_from_query_params';
 
 const AlertDetailsOverviewComponent = memo(() => {
+  const history = useHistory();
```

Comment:
:information_source: `useHistory` looks cool. I have to read about this one more.

Reply:
https://reacttraining.com/react-router/web/api/Hooks